src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj

```diff
@@ -57,8 +57,8 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.Common.Authentication, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.3.3-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
-      <Private>True</Private>
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.4.1-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
```

Comment:
This package with version `1.4.1` looks like its not published on NuGet, please make sure to:
- Have the change in https://github.com/Azure/azure-sdk-for-net/tree/AutoRest (this is AutoRest branch)
- Publish the package from there
- Update PowerShell PR to use the one published on NuGet